sympy/physics/vector/frame.py

```diff
@@ -260,7 +260,9 @@ def _w_diff_dcm(self, otherframe):
         from sympy.physics.vector.functions import dynamicsymbols
         dcm2diff = self.dcm(otherframe)
         diffed = dcm2diff.diff(dynamicsymbols._t)
-        angvelmat = diffed * dcm2diff.T
+        # angvelmat = diffed * dcm2diff.T
+        # This one seems to produce the correct result when I checked using Autolev.
+        angvelmat = dcm2diff*diffed.T
```

Comment:
This should be in a separate PR with a specific test for it. What is the test case that is broken?

Reply:
I shall add a test for it.
There was no test case for this to begin with so I didn't break anything.

Reply:
This change should not have been merged. Just wanted to note that this broke working code as seen here: https://github.com/sympy/sympy/issues/16824. This PR seemed to have been merged without my concern addressed.